### PR TITLE
quick spelling updates for metering

### PR DESCRIPTION
**Pull request by lamek**

### CODE REVIEW

**Comment by lamek** [comment]
@timflannagan1 please let me know if you saw anything else. Thanks.

File: modules/metering-cluster-usage-examples.adoc [hunk 1]
```diff
@@ -24,4 +24,4 @@ spec:
 <1> To say organized, remember to change the name of your Report if you change any of the other values.
```
Comment (timflannagan1) [inline modules/metering-cluster-usage-examples.adoc]: s/say/stay

Comment (timflannagan1) [reply modules/metering-cluster-usage-examples.adoc]: This also needs to be updated in the "Measure cluster utilization using cron expressions" section too.

**Comment by timflannagan1** [comment]
In "Measure cluster utilization using cron expressions":

You can also use cron expressions when configuring the period of your repots. The following report measure cluster utilization by looking at CPU utilization from 9am-5pm every weekday.

s/repots/reports

**Pull request merged**